src/components/AddUsersModal/index.jsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm, SubmissionError } from 'redux-form';
+import { Button, Icon, Modal } from '@edx/paragon';
+
+import H3 from '../H3';
+import TextAreaAutoSize from '../TextAreaAutoSize';
+import StatusAlert from '../StatusAlert';
+import FileInput from '../FileInput';
+import {
+  validateEmailTemplateFields,
+  validateEmailAddresses,
+  validateEmailAddressesFields,
```

Comment:
What is the difference between the existing `validateEmailTemplateFields` and the new `validateEmailAddresses`, `validateEmailAddressesFields`?

Reply:
existing `validateEmailTemplateFields` validates the email template form fields, while the new methods validate email address form fields. I created these two methods to avoid code duplication. 

`validateEmailAddressesFields` method works like `validateEmailTemplateFields` and validates the email addresses form fields. While the other `validateEmailAddresses` takes in a list of emails and returns which are valid and which are not.
I have also added comments before the function to explain each function's functionality.